build/SharpGLTF.CodeGen/SharpGLTF.CodeGen.csproj

```diff
@@ -28,6 +28,51 @@
     <None Update="Schemas\EXT_mesh_features\schema\mesh.primitive.EXT_mesh_features.schema.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\class.property.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\class.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\definitions.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\enum.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\enum.value.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\EXT_structural_metadata.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\glTF.EXT_structural_metadata.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\mesh.primitive.EXT_structural_metadata.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Update="Schemas\EXT_structural_metadata\schema\propertyAttribute.property.schema.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>

```

Comment:
This is quite verbose, I would change this to a single line:

```xml
<None Update="Schemas\EXT_structural_metadata\**\*" CopyToOutputDirectory="PreserveNewest" />
```

Reply:
Fixed in https://github.com/vpenades/SharpGLTF/pull/206/commits/13fd9d7cadba6b7a25dcdfd05da18e9ee7037248